core/app/modules/kit-library/assets/js/components/elementor-loading.js

```diff
@@ -0,0 +1,17 @@
+export default function ElementorLoading() {
+	return (
+		<div className="elementor-loading">
+			<div className="elementor-loader-wrapper">
+				<div className="elementor-loader">
+					<div className="elementor-loader-boxes">
+						<div className="elementor-loader-box"/>
+						<div className="elementor-loader-box"/>
+						<div className="elementor-loader-box"/>
+						<div className="elementor-loader-box"/>
+					</div>
+				</div>
+				<div className="elementor-loading-title">{ __( 'Loading', 'elementor' ) }</div>
+			</div>
+		</div>
```

Comment:
It should be a single Div 🤮 

```suggestion
		<div className="elementor-loader"</div
		// bunch of CSS here
```

Reply:
I just copy and paste from another place.
When your branch will be merged we will replace it.

Reply:
It won't be merged in the near future, that was for fun 😛